test/test-dbPsychologists.js

```diff
@@ -129,41 +130,14 @@ describe('DB Psychologists', () => {
     });
   });
 
-  describe('addFrenchLanguageIfMissing', () => {
-    const addFrenchLanguageIfMissing = dbPsychologists.__get__('addFrenchLanguageIfMissing');
-    it('should add french if missing with one language', async () => {
-      addFrenchLanguageIfMissing('espagnol').should.equal('Français, espagnol');
-    });
-
-    it('should add french if nothing there', async () => {
-      addFrenchLanguageIfMissing('').should.equal('Français');
-    });
-
-    it('should add french if empty spaces for languages', async () => {
-      addFrenchLanguageIfMissing('    ').should.equal('Français');
-    });
-
-    it('should not add french if already there', async () => {
-      addFrenchLanguageIfMissing('français, italien').should.equal('français, italien');
-    });
-
-    it("should not add french if 'francais' is there", async () => {
-      addFrenchLanguageIfMissing('francais').should.equal('francais');
-    });
-
-    it('should not add french (capitalized) if already there', async () => {
-      addFrenchLanguageIfMissing('Français et espagnol').should.equal('Français et espagnol');
-    });
-  });
-
   describe('getActivePsychologists', () => {
     it('should only return not archived and accepte psychologists with capitalized lastName', async () => {
       const activePsy = clean.getOnePsy();
       const archivedPsy = clean.getOnePsy('archived@psy.fr');
       archivedPsy.archived = true;
       archivedPsy.lastName = 'ArchivedPsy';
       const constructionPsy = clean.getOnePsy('construction@psy.fr');
-      constructionPsy.state = 'en_construction';
+      constructionPsy.state = 'enConstruction';
```

Comment:
Either use `en_construction` or `DossierState.enConstruction` (same for another test below)